src/instrument.js

```diff
@@ -1,6 +1,7 @@
 const APIM_CONTRACT_NAME = 'manager.apim'
 const INSTR_CONTRACT_NAME = 'instr.ore'
-const INSTR_TABLE_NAME = 'tokens'
+const INSTR_USAGE_CONTRACT_NAME = 'usagelog.ore'
+const TABLE_NAME = 'instruments'
```

Comment:
Are we using this anywhere?

Reply:
not using "instruments" table 
using "tokens" table to get all the instruments